browser/ui/views/brave_default_browser_dialog_view.cc

```diff
@@ -10,12 +10,17 @@
 #include "base/bind.h"
 #include "base/memory/scoped_refptr.h"
 #include "brave/browser/ui/browser_dialogs.h"
+#include "brave/common/pref_names.h"
 #include "brave/grit/brave_generated_resources.h"
+#include "chrome/browser/browser_process.h"
```

Comment:
Need to add a dep on `//chrome/browser:browser_process`?

Reply:
Fixed at https://github.com/brave/brave-core/pull/8253/files#diff-7ce4cface9f839816f0fc02bad0e04aa80284b048ef0d8f0876547e6e37e38b6R253